examples/source_file_from_fetched_data.html

```diff
@@ -70,9 +70,9 @@
 
         view.addLayer(new itowns.ColorLayer('Roads', {
             source: sourceFromFetchedData,
-            style: new itowns.Style({
+            style: {
                 stroke: { color: 'yellow' },
-            }),
```

Comment:
Remove instancing, it's good move. 
This change alone can be in another PR. 
This would simplify the review and see if there are any side effects.

It's still possible to put an instance of Style as an option?

Reply:
It's still possible to put an instance of Style as an option but in this case (as it was before (in the cases where it was working)) you won't be able to get the missing style properties from the data metadata.
Removing the instancing can't really be done in another PR as you still need to instance it in the convert, for steing the default values)